packages/trader/src/Modules/CFD/Containers/cfd-password-modal.tsx

```diff
@@ -70,11 +192,12 @@ const PasswordModalHeader = ({
         </Text>
     );
 };
-const getSubmitText = (type, category, platform, is_eu, needs_poi) => {
+const getSubmitText = (platform: string, is_eu: boolean, needs_poi: boolean, type?: string, category?: string) => {
```

Comment:
I had to change the order of the args here and at line 952 where the function is used in order to avoid an error: "A required parameter cannot follow an optional parameter"